SeleniumTestsWithoutPOM3/homework2.cs

```diff
@@ -0,0 +1,80 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+
+namespace SeleniumTestsWithoutPOM3
+{
+    internal class homework2
+    {
+        // nepavykes
+        [Test]
+        public void TextBoxTest()
+        {
+            IWebDriver driver = new ChromeDriver();
+            driver.Manage().Window.Position = new System.Drawing.Point(2000, 1);
+            driver.Manage().Window.Maximize();
+
+            driver.Url = "https://demoqa.com/text-box";
+
+            string expectedFullName = "Ieva Montrimaite";
+            string expectedEmail = "montrimaite.ieva@gmail.com";
+            string expectedCurrentAddress = "2436 N 48th St #101, Lincoln, New Hampshire";
+            string expectedPermanentAddress = "8602 SW State Road 200, Ocala, Florida";
+
+            IWebElement inputFullName = driver.FindElement(By.XPath("//*[@id='userName']"));
+            IWebElement inputEmail = driver.FindElement(By.XPath("//*[@id='userEmail']"));
+            IWebElement inputCurrentAddress = driver.FindElement(By.XPath("//*[@id='currentAddress']"));
+            IWebElement inputPermanentAddress = driver.FindElement(By.XPath("//*[@id='permanentAddress']"));
+            IWebElement buttonSubmit = driver.FindElement(By.XPath("//*[@id='submit']"));
+
+            inputFullName.SendKeys(expectedFullName);
+            inputEmail.SendKeys(expectedEmail);
+            inputCurrentAddress.SendKeys(expectedCurrentAddress);
+            inputPermanentAddress.SendKeys(expectedPermanentAddress);
+            buttonSubmit.Click();
+
+            IWebElement spanMessageName = driver.FindElement(By.XPath("//*[@id='name']"));
+            IWebElement spanMessageEmail = driver.FindElement(By.XPath("//*[@id='email']"));
+            IWebElement spanMessageCurrentAddress = driver.FindElement(By.XPath("//*[@id='currentAddress']"));
+            IWebElement spanMessagePermanentAddress = driver.FindElement(By.XPath("//*[@id='permanentAddress']"));
+
```

Comment:
Niekaip nepavyko išgauti įvesto teksto xpath: 

- //*[@id='names'/text()[2]] randa konkretų elementą puslapyje, bet Seleniumas tokio nepriima, meta error'ą: 

_"//*[@id='names'/text()[2]]" is: [object Text]. It should be an element._

- bandžiau žaisti su contains, tačiau man rado tik pilną elementą su id, o ne reikiamą teksto dalį, pvz:
//*[text()[contains(.,'Ieva Montrimaite')]]
//p[contains(., 'Ieva Montrimaite')]